Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -406,11 +406,21 @@ private final class TokenStreamCreator: SyntaxVisitor {
     super.visit(node)
   }
 
+  override func visit(_ node: CodeBlockItemListSyntax) {
+    if node.parent is AccessorBlockSyntax {
+      for i in 0..<(node.count - 1) {
```

Comment:
We need to guard against `node.count == 0` here, right? Such an accessor wouldn't be valid in the language (it needs to return something) but we don't want to crash if someone does it.